indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -898,6 +899,7 @@ public boolean isAppendToExisting()
     private static final boolean DEFAULT_GUARANTEE_ROLLUP = false;
     private static final boolean DEFAULT_REPORT_PARSE_EXCEPTIONS = false;
     private static final long DEFAULT_PUBLISH_TIMEOUT = 0;
+    private static final Period DEFAULT_LOCK_TIMEOUT = new Period("PT5m");
```

Comment:
@jihoonson I suggest to use HttpClient timeout as default lock timeout. Say if http client timeout is less than 5 minutes, client anyways can not get the response back even after waiting for 5 minutes.